features/adres.feature

```diff
@@ -1,96 +1,90 @@
-      | regel 3 adres buitenland (13.50)  | postcode 1000   |
-      En element woonplaatsnaam (11.70) heeft geen waarde
-      Als persoon met burgerservicenummer "{burgerservicenummer}" wordt opgevraagd met fields=verblijfplaats.woonplaats,verblijfplaats.gemeenteVanInschrijving
-      Dan bevat in het antwoord de verblijfplaats GEEN veld "straat"
-      En bevat in het antwoord de verblijfplaats GEEN veld "korteNaam"
 
   Rule: Voor een binnenlands adres wordt veld "woonplaats" gevuld met de woonplaatsnaam (11.70) wanneer die bekend is, en anders met de omschrijving van de gemeente van inschrijving (09.10 via tabel 33 Gemeententabel).
 
     Abstract Scenario: opnemen woonplaats binnenlands adres
-      Gegeven persoon met burgerservicenummer "{burgerservicenummer}" heeft de volgende gegevens bij verblijfplaats:
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde                               |
+      | identificatie       | 4cbcf7fd-d788-4be5-8ebc-17decde26131 |
+      | burgerservicenummer | <burgerservicenummer>                |
+      En de persoon heeft de volgende verblijfplaats gegevens
+      | naam                              | waarde           |
       | gemeente van inschrijving (09.10) | 0518             |
-      | woonplaatsnaam (11.70)            | {woonplaatsnaam} |
-      Als persoon met burgerservicenummer "{burgerservicenummer}" wordt opgevraagd met fields=verblijfplaats.woonplaats,verblijfplaats.gemeenteVanInschrijving
-      Dan bevat in het antwoord verblijfplaats:
-      """
-      {
-        "gemeenteVanInschrijving": {
-            "code": "0518",
-            "omschrijving": "'s-Gravenhage"
-        },
-        "woonplaats": "{woonplaats}"
-      }
-      """
+      | woonplaatsnaam (11.70)            | <woonplaatsnaam> |
+      En de waardetabel 'Gemeenten' heeft de volgende waarden
+      | code | omschrijving  |
+      | 0518 | 's-Gravenhage |
+      Als personen wordt geraadpleegd met de volgende parameters
+      | naam                 | waarde                                                           |
+      | persoonIdentificatie | 4cbcf7fd-d788-4be5-8ebc-17decde26131                             |
+      | fields               | verblijfplaats.woonplaats,verblijfplaats.gemeenteVanInschrijving |
+      Dan bevat de persoon met burgerservicenummer '<burgerservicenummer>' de volgende verblijfplaats gegevens
+      | naam                                 | waarde        |
+      | gemeenteVanInschrijving.code         | 0518          |
+      | gemeenteVanInschrijving.omschrijving | 's-Gravenhage |
+      | woonplaats                           | <woonplaats>  |
 
       Voorbeelden:
       | burgerservicenummer | woonplaatsnaam | woonplaats    |
       | 999992922           | Scheveningen   | Scheveningen  |
       | 999990639           |                | 's-Gravenhage |
 
-    Scenario: persoon verblijft in het buitenland
```

Comment:
@MelvLee waarom is dit scenario vervallen?

Reply:
woonplaatsnaam komt niet voor als property in verblijfplaatsbuitenland en gemeenteVanInschrijving wordt niet gevalideerd in de Dan stap. Dan heeft deze scenario volgens mij geen waarde? Het valideren van gemeenteVanInschrijving heb ik nu toegevoegd bij de andere 'persoon verblijft in het buitenland' scenario onder rule 'Voor een buitenlands adres worden de adresregels en land gevuld uit de gegevens over verblijf buitenland (groep 13)'

Reply:
het doel van dit scenario is te tonen dat bij een verblijf buitenland (waarbij er geen woonplaats is gevuld in de bron) de woonplaats niet wordt geleverd/gevuld met gemeenteVanInschrijving.omschrijving, zoals wel gebeurt bij een binnenlands adres of locatiebeschrijving waarbij er geen woonplaats is gevuld in de bron